components/ws-manager-bridge/src/workspace-instance-controller.ts

```diff
@@ -204,22 +197,19 @@ export class WorkspaceInstanceControllerImpl implements WorkspaceInstanceControl
     async controlNotStoppedAppClusterManagedInstanceTimeouts(
         parentCtx: TraceContext,
         runningInstances: RunningWorkspaceInfo[],
-        applicationClusterName: string,
     ) {
         const span = TraceContext.startSpan("controlNotStoppedAppClusterManagedInstanceTimeouts", parentCtx);
         const ctx = { span };
         try {
-            log.debug("Controlling app cluster managed instances...", { applicationClusterName });
```

Comment:
This distinction is actually important. I see no reason to remove it.

Reply:
But the value it contains is always going to be `us-02`, no?

Reply:
I mean the "Controlling **app cluster** managed instances..."

Reply:
I agree on the applicationClusterName, that we can ditch. :+1: